stable/firehose/templates/_helpers.tpl

```diff
@@ -46,7 +46,8 @@
   delete_sets = true
   delete_timings = true
   metric_separator = "."
-  parse_data_dog_tags = true
+  parse_data_dog_tags = false
```

Comment:
can we remove this key? as it is going to be deprecated?

Reply:
fixed